src/devices/DCMotorController/DCMotorController.cs

```diff
@@ -0,0 +1,96 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+using System.Device.Pwm;
+using System.Device.Pwm.Drivers;
+
+namespace Iot.Device
```

Comment:
Apart from changing the DCMotor name to an actual device that is supported by this binding, we need to include that class name as part of the namespace since all of the drivers go into one bindings dll where we keep them apart by using namespaces.

Reply:
DCMotorController works the same for all motors and the only difference is how you connect them but I'm planning to handle this in this class. With Servo I believe we can have both specific and generic version since Servo has some more params